src/scss/buttontabs.scss

```diff
@@ -7,10 +7,6 @@
 
 		[role=tab] {
 			@include oButtons;
-
-			a {
-				pointer-events: none;
```

Comment:
How come this was removed?

Reply:
Needs to receive a pointer event to follow the link.